### ISSUE TITLE
Storing options in database ?

### ISSUE THREAD

**Issue by amadeous** [bfabiszewski/ulogger-server#10]
Hi !

It would be nice to be able to change map center during set up for example and in an administration panel later on if needed.

Ring me if I can help

**Comment by bfabiszewski** [comment]
Do you think that setting map center in the config file is not enough? 
Is there any reason to change this setting later on? Generally this is only visible initially, when there are no routes.
If we wanted to change it during setup or later on, we would have to store the settings in database, so it complicates things a bit.

**Comment by amadeous** [comment]
You're right. I haven't notice that in the last updates.
Sorry, my bad.
Why not storing the option in database ? Language options for example ?

**Comment by bfabiszewski** [comment]
This is something we could think of in a future version.

It would probably mean adding some kind of settings screen. Till now I tried to put everything on a single page, but it may not fit with more options. Initial settings could be chosen in a setup script.

There should be general app settings and also user settings for each logged in user, like e.g. language (currently stored only in cookies).

On the other hand I like to keep things simple. Anyway let's wait for some more feedback. I mean, if there are only a few people using this project, there is no reason to make things more complicated :)

**Comment by amadeous** [comment]
I get your point !
Maybe a whole page for settings could be more plaisant and still simple don't you think ?

What about extending the chart for altitude to speed, acceleration ?

**Comment by bfabiszewski** [comment]
- Regarding separate settings page, this is probably the way to go if the application is to be developed further.
- Charts for speed and acceleration would be good. The thing that stopped me with further improving speed statistics was that many devices register wrong speed (often zero), making any statistics useless. Maybe the viewer could use calculated speed instead.
- The idea with colored tracks to show different speed (or other parameters) is very nice. I've already thought about it, but didn't have time yet. I'll get back to it.
- The interval for registration. If you mean on the client (android) side, that's exactly was I was planning to add if time allows – presets for different types of activities.

Thanks for all your contributions and spreading the word!

**Comment by Lebewesen** [comment]
Very nice points - I think calculating the speed is better than the speed at one point. I hacked the speed chart for me where I used the speed of each point - it was good for an overview - but on this track I was on a bike, and i did a run. It didn't show on the graph because the points where to wide spaced, and the the top speed was between to points. I think a graph with points showing the speed at this point is a start. But I don't want to repace it with a calulated speed histogram. I thing there should be a toggle to switch between both views.
